README.md

```diff
@@ -1,2 +1,83 @@
 # Reps-Mobile
-reps.mozilla.org data on mobile
+
+<p align="center">
+  <a href="https://flutter.dev/">
+    <img alt="Flutter" src="https://flutter.dev/assets/flutter-lockup-4cb0ee072ab312e59784d9fbf4fb7ad42688a7fdaea1270ccf6bbf4f34b7e03f.svg" width="170" />
+  </a>
+</p>
+<h1 align="center">
+ Flutter Starter 
+</h1>
+
+Kick off your project with this boilerplate. This starter will help to install  the main flutter configuration files along with the brief introduction to Mozilla Reps mobile application.
+
+## 📝 Description
+
+    reps.mozilla.org data on mobile
```

Comment:
This does not need to be in quotations. Please remove them
Thanks

Reply:
The ones in Bold? Also Should remove Each one of them?

Reply:
Just keep one link.